src/components/tools/ManipulatorTool.vue

```diff
@@ -38,38 +39,79 @@ export default defineComponent({
         : viewProxy.value.getInteractorStyle3D();
     });
 
-    const manipulator = props.manipulatorClass.newInstance(options.value);
+    const manipulators: any[] = [];
 
-    watch(options, (newOptions) => {
-      if ('button' in newOptions) {
-        manipulator.setButton(newOptions.button);
-      }
-      if ('shift' in newOptions) {
-        manipulator.setShift(newOptions.shift);
-      }
-      if ('control' in newOptions) {
-        manipulator.setControl(newOptions.control);
+    function updateStyle(
+      style: any,
+      newManipulators: any[],
+      oldManipulators: any[]
+    ) {
+      if (style !== null) {
+        oldManipulators.forEach((m) => {
+          if (m.isA('vtkCompositeMouseManipulator')) {
```

Comment:
I don't think we need these type checks, since if someone passes in a non-mouse manipulatorClass, the watch block would throw an error when setting the options anyways.

Reply:
Remove the `isA` checks